tests/cbor/tests/01-run.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2017 OTA keys S.A.
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+
+sys.path.append(os.path.join(os.environ['RIOTBASE'], 'dist/tools/testrunner'))
+import testrunner
+
+ACCEPTED_ERROR = 20
+
+def testfunc(child):
+    child.expect(r'''Data:\r
```

Comment:
I find that very hard to read. Can you do a `child.expect()` for every output line, please?